packages/commonwealth/client/scripts/views/pages/Editor/DesktopEditorFooter.tsx

```diff
@@ -0,0 +1,65 @@
+import { Clipboard, DownloadSimple } from '@phosphor-icons/react';
+import React, { useCallback, useRef } from 'react';
+import './DesktopEditorFooter.scss';
+
+type DesktopEditorFooterProps = Readonly<{
+  onImportMarkdown?: (file: File) => void;
+  onSubmit?: () => void;
+}>;
+
+export const DesktopEditorFooter = (props: DesktopEditorFooterProps) => {
+  const { onImportMarkdown, onSubmit } = props;
+  const fileInputRef = useRef<HTMLInputElement | null>(null);
+
+  const fileHandler = useCallback(
+    (event: React.ChangeEvent<HTMLInputElement>) => {
+      if (event.target.files) {
+        console.log('nr files: ' + event.target.files.length);
+        onImportMarkdown?.(event.target.files[0]);
+      }
+    },
+    [onImportMarkdown],
+  );
+
+  const handleImportMarkdown = useCallback(() => {
+    if (fileInputRef.current) {
+      fileInputRef.current.click();
+    }
+  }, []);
+
+  return (
+    <div className="DesktopEditorFooter">
+      <div className="Item">
+        <div className="IconAndText">
+          <div>
+            <Clipboard size={24} />
+          </div>
+          <div>Paste, drop or click to add files</div>
+        </div>
+      </div>
+      <div className="Item">
+        <input
+          ref={fileInputRef}
+          type="file"
+          accept=".md"
+          className="FilePickerInput"
+          onChange={fileHandler}
+        />
```

Comment:
the validation is not enforced here - the user can still select non-markdown files via the file selector - lets add forced validation to ensure only .md files get through

Reply:
I will check but I don't think they can... At least I tested that in the past.

Reply:
@mzparacha There's now a .md extension validation and we ignore non-markdown files in the code.

Note that MacOS filters the file input and so does Windows (pretty sure). Are you on Linux?

Reply:
Actually its still working, i am able to upload a `.control` file and a `.csv` file, which were imported successfully. I get a UI error with `.json`, `.sql` files. Didn't test with others (like media files) but I feel some other files will also get through without any errors.

https://github.com/user-attachments/assets/a5a07aa5-1f4f-421f-bb0d-4b66176728c7

Reply:
Nice. I didn't know you could bypass that!  Anyway. That's fixed now.